gallery/src/styles/NxList/NxListMultiLineExample.html

```diff
@@ -34,7 +34,7 @@ <h3 class="nx-h3">
     <span class="nx-list__text">Subtext truncation</span>
     <p class="nx-list__subtext nx-truncate-ellipsis">
       Add <code class="nx-code">.nx-truncate-ellipsis</code> to <code class="nx-code">.nx-list__subtext</code> to
-      cause the text to truncate at the standard maximum paragraph width.
+      cause the text to truncate at the right edge of the list item.
```

Comment:
This isn't long enough to truncate on my system

Reply:
> This isn't long enough to truncate on my system

Should be now: https://github.com/sonatype/sonatype-react-shared-components/pull/141/commits/deecaa9b411f3a66bd96422ed0202be0fbde20cb